learnosity_sdk/request.py

```diff
@@ -219,3 +222,62 @@ def set_service_options(self):
     def hash_list(self, l):
         "Hash a list by concatenating values with an underscore"
         return hashlib.sha256("_".join(l).encode('utf-8')).hexdigest()
+
+
+class DataApi(object):
```

Comment:
I can do that. I didn't because to make it consistent I would need to move Init into its own file too and modify __init__.py to expose both classes, and I wanted to avoid changing Init too much.

Reply:
Yeah, it's fine. I wouldn't move `Init` anywhere for now, but just avoid multiplicating the same antipattern.